polyaxon/templates/ing.yaml

```diff
@@ -17,6 +17,9 @@ metadata:
   annotations:
 {{ toYaml .Values.ingress.annotations | indent 4 }}
 spec:
+  {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
I think this is only supported by helm [v2](https://v2.helm.sh/docs/chart_template_guide/#built-in-objects), in [v3](https://helm.sh/docs/chart_template_guide/builtin_objects/) there's no `GitVersion`